services/core-web/src/components/common/CoreTooltip.js

```diff
@@ -26,7 +27,13 @@ export const CoreTooltip = (props) => (
 );
 
 export const NOWFieldOriginTooltip = () => (
-  <CoreTooltip title="This field is not being sent by NROS or vFCBC. Open the original PDF to see the data." />
+  <Tooltip
```

Comment:
Why no longer use the CoreTooltip here?

Reply:
I didn't want to change the Icon for all of the CoreTooltips